src/docker.test.ts

```diff
@@ -86,7 +86,9 @@ describe("Docker images", (): void => {
 
```

Comment:
Commit message (nit): "was being run in parallel for multiple jobs" --> "was run in parallel" for brevity